packages/wrangler/src/api/pages/deploy.tsx

```diff
@@ -295,7 +295,8 @@ export async function deploy({
 
 	if (_workerJS || _workerJSIsDirectory) {
 		const workerBundleContents = await createUploadWorkerBundleContents(
-			workerBundle as BundleResult
+			workerBundle as BundleResult,
+			undefined
```

Comment:
We are not passing config in here because the config will already be stored in the `workerBundle`?

Reply:
This file will be addressed in [a followup ](https://jira.cfdata.org/browse/DEVX-1176)